sdk/key_vault/src/key.rs

```diff
@@ -239,6 +239,54 @@ impl Display for SignatureAlgorithm {
     }
 }
 
+#[derive(Debug, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct DecryptResult {
+    aad: Option<String>,
+    iv: Option<String>,
+    #[serde(rename = "kid")]
+    key_id: String,
+    tag: Option<String>,
+    #[serde(serialize_with = "ser_base64", deserialize_with = "deser_base64")]
+    value: Vec<u8>,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum EncryptionAlgorithm {
```

Comment:
Please keep consistent casing. If it's idiomatic for all-uppercase characters, do so throughout. See https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/EncryptionAlgorithm.cs for how .NET did this. IIRC, Java is all uppercase since that is idiomatic for Java statics.

Reply:
I will follow this as well. But I was looking at the SignatureAlgorithms above, which doesn't follow the .NET.

Reply:
At this point we're not concerned with breaking changes - in fact, there will be a *lot* more to come - so you could make `SignatureAlgorithms` match the guidance or ignore for now and I'll clean it up later.